### DISCUSSION TITLE
Elliptical mask

### DISCUSSION THREAD

**Discussion by Ferr013** [ajshajib/dolphin#45] [Q&A] [answered]
Hi,

is it possible to create elliptical masks?
I saw there is a way to pass `custom_mask` in the `settings.yml` file as pixel by pixel booleans,
but for the images I am working with (400x400) this leads to very (very) long `setting.yml` files that become a bit clunky to scroll and modify. Also, these large files seem to crash IPython kernels.

**Comment by ajshajib** [comment, marked as answer]
Thanks a lot for asking about it. The feature did not exist, but I added it through #46. To create elliptical masks, you can simply pass lists of "a," "b," and "angle" instead of "radius".

**Reply by Ferr013** [reply]
Awesome thank you @ajshajib!
